tuda_workspace_scripts/discovery.py

```diff
@@ -0,0 +1,63 @@
+from robots import ZenohRouter, load_robots
+from tuda_workspace_scripts import print_warn
+import os
+import yaml
+
+CONFIG_DIR = os.path.expanduser("~/.config")
```

Comment:
Not sure if I would just overwrite any existing configs here.
Can't we configure zenoh to use a different directory for the config?
E.g. in our workspace?